plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/ray_launcher_util.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+from typing import Any, Dict
+
+import ray
+from omegaconf import DictConfig
+from ray._raylet import ObjectID
+
+from hydra import TaskFunction
+from hydra.core.config_search_path import ConfigSearchPath
+from hydra.core.hydra_config import HydraConfig
+from hydra.core.singleton import Singleton
+from hydra.core.utils import run_job, setup_globals
+from hydra.plugins.search_path_plugin import SearchPathPlugin
+
+log = logging.getLogger(__name__)
+
+JOB_SPEC_PICKLE = "job_spec.pkl"
+JOB_RETURN_PICKLE = "returns.pkl"
+
+
+class RayLauncherSearchPathPlugin(SearchPathPlugin):
+    def manipulate_search_path(self, search_path: ConfigSearchPath) -> None:
+        # Appends the search path for this plugin to the end of the search path
+        search_path.append(
+            "hydra-ray-launcher", "pkg://hydra_plugins.hydra_ray_launcher.conf"
```

Comment:
you should probably add a config search path for 
`structured://hydra_plugins.hydra_ray_launcher.config`

where config is the module that is registering your structured configs.

you also no longer have a conf module, but I am proposing that we add it back so don't delete it just yet).